src/Bootstrappers/FilesystemTenancyBootstrapper.php

```diff
@@ -84,14 +83,9 @@ public function revert()
         $this->app['url']->setAssetRoot($this->app['config']['app.asset_url']);
 
         // Storage facade
+        Storage::forgetDisk($this->app['config']['tenancy.filesystem.disks']);
```

Comment:
Is `forgetDisk()` used to solve the prefix issue without having to manipulate the existing disk instances?

Reply:
Correct, it makes sure that the next time the disk is accessed, it is recreated from the new config.